### ISSUE TITLE
Program Only Handles .bmp Images

### ISSUE THREAD

**Issue by lordmauve** [lordmauve/pgzero#14]
Originally reported by: **Steve Holden (Bitbucket: [holdenweb](https://bitbucket.org/holdenweb), GitHub: [holdenweb](https://github.com/holdenweb))**

----------------------------------------

Despite the many examples that use .png images I discovered that the program complains when images are anything other than a Windows Bitmap.

----------------------------------------
- Bitbucket: https://bitbucket.org/lordmauve/pgzero/issue/14

**Comment by lordmauve** [comment]
*Original comment by* **Arve Seljebu (Bitbucket: [arve0](https://bitbucket.org/arve0), GitHub: [arve0](https://github.com/arve0))**:

----------------------------------------

If you are on mac, you'll need to install `sdl_image`, then reinstall `pygame`:

```sh
brew install sdl_image
pip install -U hg+http://bitbucket.org/pygame/pygame
```

**Comment by lordmauve** [comment]
*Original comment by* **Daniel Pope (Bitbucket: [lordmauve](https://bitbucket.org/lordmauve), GitHub: [lordmauve](https://github.com/lordmauve))**:

----------------------------------------

Resolved by pull request #10. Existing installations will have to be upgraded using the above instructions.

**Comment by lordmauve** [comment]
*Original comment by* **Daniel Pope (Bitbucket: [lordmauve](https://bitbucket.org/lordmauve), GitHub: [lordmauve](https://github.com/lordmauve))**:

----------------------------------------

Issue #9 was marked as a duplicate of this issue.
